exercises/060_floats.zig

```diff
@@ -26,7 +27,10 @@
 // operations with numeric literals, ensure the types match. Zig
 // does not perform unsafe type coercions behind your back:
 //
-//     var foo: f16 = 13.5 * 5;   // ERROR!
+//     fn foo(bar: u16) f16 { return 13.5 * bar; } // ERROR!
+//     var foo: f16 = 13.5 * @as(u8, 5); // ERROR!
+//     var foo: f16 = 13.5 * 5; // This is a safe compile-time
+//                              // conversion, so no problem!
```

Comment:
Yeah, those examples certainly work...  But I played around a bit and realized that we can actually get the original behavior with two variables (`float_var = int_var` produces the original error).

I think the multiplication is just getting in the way of the example now.

But you're right, it _would_ be nice to also demonstrate the compile-time conversion. So how about this to demonstrate the difference:

```
    var u_foo: u16 = 5;
    var float_foo: f16 = u_foo; // ERROR! (Runtime type conversion.)
    var float_foo: f16 = 5; // Fine! (Safe compile-time numeric evaluation.)
```

What do you think?

Reply:
Yes, I think we agree!

I responded to the commit before I saw your comment. I like the change.

But I would like to move away from using `@as()` in the example (see code review comment).